### PR TITLE
Fixes #12203: Interface to manage ACL and user token for API authorization

### PR DESCRIPTION
**Pull request by fanf**
https://www.rudder-project.org/redmine/issues/12203

### CODE REVIEW

**Comment by fanf** [comment]
PR rebased

File: api-authorizations/pom-template.xml [hunk 1]
```diff
@@ -27,9 +30,71 @@
...
+            <phase>process-resources</phase>
+            <goals>
+              <goal>copy-resources</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>target/classes/template</outputDirectory>
+              <resources>
+                <resource>
+                  <directory>src/main/elm</directory>
+                  <includes><include>reportingManagement.html</include></includes>
```
Comment (VinceMacBuche) [inline api-authorizations/pom-template.xml]: I think this file is missing and it should be renamed according to correct plugin name

Comment (VinceMacBuche) [reply api-authorizations/pom-template.xml]: In fact we do not need that section about the html template we will include an plugin on api account page

File: api-authorizations/src/main/scala/com/normation/plugins/apiauthorizations/snippet/ServeResource.scala [hunk 2]
```diff
@@ -35,7 +35,7 @@
 *************************************************************************************
```
Comment (VinceMacBuche) [inline api-authorizations/src/main/scala/com/normation/plugins/apiauthorizations/snippet/ServeResource.scala]: ServeResource.scala should be delete

**Comment by VinceMacBuche** [comment]
This needs to be rebased

**Comment by fanf** [comment]
PR rebased

**Comment by fanf** [comment]
OK, merging this PR

**Pull request merged**